include/envoy/http/header_map.h

```diff
@@ -440,6 +440,15 @@ class HeaderMap {
    */
   virtual void addCopy(const LowerCaseString& key, const std::string& value) PURE;
 
+  /**
+   * Appends data to header. If header already has a value, the string ',' is added between the
+   * existing value and data.
+   * @param key specifies the name of the header to append.
+   * @param value specifies the value of the header to add. If the string is a reference string the
+   * reference data is not copied. This uses the underlying HeaderString::append method.
+   */
+  virtual void appendToHeader(const LowerCaseString& key, const std::string& value) PURE;
```

Comment:
string_view instead of `std::string&`

Reply:
Done! Does the `LowerCaseString` construction make sense? I was a little stumped on this one. I felt the `string_` should be kept a string, but wasn't sure about the explicit construction from a string.

Reply:
Sorry what do you mean?